setup-nexus-npm/README.md

```diff
@@ -1,22 +1,20 @@
 # setup-nexus-npm
 
-Configure a github action to point towards a self hosted Npm repository. After usage, any calls to `npm install` will point towards the repository at the endpoint provided.
+A composite github action for creating a .npmrc file that will point to the npm repositories in nexus
```

Comment:
Perhaps we should include a blurb as to what this allows folks to do ie "allowing developers to use Nexus to both access TXM internal Node modules as well as as a proxy for all Node dependencies"

Reply:
added